src/main/java/org/alliancegenome/curation_api/interfaces/crud/ontology/ZfaTermRESTInterface.java

```diff
@@ -0,0 +1,18 @@
+package org.alliancegenome.curation_api.interfaces.crud.ontology;
+
+import org.alliancegenome.curation_api.base.BaseCrudRESTInterface;
+import org.alliancegenome.curation_api.model.entities.ontology.ZfaTerm;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/zfaTerm")
```

Comment:
All the other endpoints this is lowercase can we make it `zfaterm` to be consistant with the rest of them?